managers/manage_objects.go

```diff
@@ -361,6 +361,86 @@ func CloneServices(clientset *kubernetes.Clientset, sourceNamespace, targetNames
 	return nil
 }
 
+func CloneIstioVirtualServices(dynamicClient dynamic.Interface, sourceNamespace, targetNamespace string) *Error {
```

Comment:
This implementation is in-complete. For instance, you cannot clone a virutalservice in this case, instead should create a new one? Why, because we want to enable something like a `feature-branch-<existingapp name>.test.slice.com`